apps/next-cypress/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Interesting learning thing - in a monorepo, we should only have a `tsconfig` at the repo root - and if absolutely necessary a lower down one that extends that.

Reply:
sad thing: Next always generates a tsconfig for you

Reply:
BOOOOOO

What even?

We can't even gitignore them because sometimes we need them right? :(

Reply:
So in the latest commit - `707e2d2` I deleted most of the auto generated Cypress tests and kept only one. I think the `.tsconfig` can be fixed, that shouldn't be an issue.